lib/logstash/inputs/s3.rb

```diff
@@ -386,7 +390,12 @@ def delete_file_from_bucket(object)
 
   private
   def get_s3object
-    s3 = Aws::S3::Resource.new(aws_options_hash)
+    options = aws_options_hash || {}
+    if @force_path_style == true
+      options.merge!(:force_path_style => @force_path_style)
```

Comment:
is this an `AWS::S3` flag ?  I can't find it documented .... do you have a link ?

Reply:
I'm thinking that probably we'd like to remove some of the current config settings as they'd be done through the `additional_settings` hash.